content/blog/2021/10/new-architecture-to-ease-kubewarden-administrators-lives.md

```diff
@@ -0,0 +1,257 @@
+
+```yaml
+---
+apiVersion: policies.kubewarden.io/v1alpha2
+kind: ClusterAdmissionPolicy
+metadata:
+  name: psp-capabilities
+spec:
+  policyServer: tenant-a
+  module: registry://ghcr.io/kubewarden/policies/psp-capabilities:v0.1.3
+  rules:
+    - apiGroups: [""]
+      apiVersions: ["v1"]
+      resources: ["pods"]
+      operations:
+        - CREATE
+        - UPDATE
+  mutating: true
+  settings:
+    allowed_capabilities:
+      - CHOWN
+    required_drop_capabilities:
+      - NET_ADMIN
+```
+
+For a more in depth dive, have a look at our [architecture
+docs](https://docs.kubewarden.io/architecture.html), and the [Kubewarden CRDs documentation](https://github.com/kubewarden/kubewarden-controller/blob/main/docs/crds/README.asciidoc).
+
+## Ok, but what does this mean for administrators?
+
+Moving from 1 PolicyServer to several increases resiliency and segregation. Now,
+we can take admission reviews from policies and separate them into several
+PolicyServers. While the old architecture was already HA, a noisy
+tenant/namespace or a frequently used policy could bring to a crawl the only
+policy server and break havoc in the Cluster. With the new possibility of
+several PolicyServers, a Kubernetes Operator can isolate policy evaluations per
+tenant/namespace, and run mission critical policies separately, making the whole
+infrastructure more resilient.
+
+The new architecture also validates and mutates PolicyServers and
+ClusterAdmissionPolicies with dedicated admission controllers, for a better UX.
+Which means that administrators can rest comfortably when editing them, as
+catastrophic outcomes (such as all policies being dropped by a misconfigured
+PolicyServer, leading to DOS against the cluster) can never happen. Also,
+ClusterAdmissionPolicies will, if no `spec.policyServer` is defined, bind to the
+PolicyServer named `default`. In addition,
+[Finalizers](https://kubernetes.io/docs/concepts/overview/working-with-objects/finalizers/)
+are now added to all Kubewarden CRs, which ensure orderly deletion by the
+Kubewarden Controller.
+
+The inclusion of validating and mutating webhooks for Kuberwarden CRDs means that
+the controller webhook server needs to be securely hooked up to the Kubernetes
+API. In this case, it means using TLS certificates. We have chosen to integrate
+Kuberwarden with [cert-manager](https://cert-manager.io/), to simplify the
+installation, and our Helm Chart today comes with the option for automatically
+creating and setting up Self-Signed certs, or using your own cert-manager
+Issuer.
+
+For ease of deployment, we have separated the CRDs into its own Helm chart:
+`kubewarden-crds`. This prepares the stack for smoother upgrades in the future.
+The Kubewarden Controller and default policy server stay in the
+`kubewarden-controller` Helm chart.
+
+All of the new changes simplify managing clusters. Which makes managing multiple
+clusters via Fleet more consistent and streamlined.
```

Comment:
Managing clusters and managing multiple clusters are together here, I think we could rephrase this bit.